sdk/storage/azure-storage-queue/samples/queue_samples_message.py

```diff
@@ -50,7 +51,7 @@ def set_access_policy(self):
             access_policy = AccessPolicy()
             access_policy.start = datetime.utcnow() - timedelta(hours=1)
             access_policy.expiry = datetime.utcnow() + timedelta(hours=1)
-            access_policy.permission = QueueSasPermissions(read=True)
+            access_policy.permission = QueueSasPermissions(read=True)  # type: ignore
```

Comment:
What are we ignoring here? Ideally, we shouldn't do any ignores in samples. It likely indicates something else is wrong somewhere.